main/views.py

```diff
@@ -38,11 +40,17 @@ def index():
     db.session.commit()
 
     return render_template(
-        "index.html",
-        images=images)
+            "index.html",
+            images=images)
 
 
-@app.route('/hello')
-def hello_again():
-    return 'Hello again!'
+@app.route('/news')
+def r_news():
+    user_agent = ("Python test app by /u/natyahlyi")
+    r = praw.Reddit(user_agent=user_agent)
```

Comment:
too short name, I suggest to use short ones only for loops (`for i in ....`) etc